test/extended/util/framework.go

```diff
@@ -237,23 +237,13 @@ func DumpPodLogs(pods []kapiv1.Pod, oc *CLI) {
 			e2e.Logf("Error retrieving description for pod %q: %v\n\n", pod.Name, err)
 		}
 
-		depOutput, err := oc.Run("logs").Args("pod/" + pod.Name).Output()
-		if err == nil {
-			e2e.Logf("Log for pod %q\n---->\n%s\n<----end of log for %[1]q\n", pod.Name, depOutput)
-		} else {
-			e2e.Logf("Error retrieving logs for pod %q: %v\n\n", pod.Name, err)
-		}
-	}
-
-}
-
-func DumpPodContainerLogs(podName string, containerNames []string, oc *CLI) {
-	for _, c := range containerNames {
-		output, err := oc.Run("logs").Args(podName, "-c", c).Output()
-		if err == nil {
-			e2e.Logf("Log for pod %q container %q\n---->\n%s\n<----end of log for %[1]q%[2]q\n", podName, c, output)
-		} else {
-			e2e.Logf("Error retrieving logs for pod %q container %q: %v\n\n", podName, c, err)
+		for _, container := range pod.Spec.Containers {
```

Comment:
will this catch init containers?  if not, this won't fix the build logs dumping issue.  (and dumping the init containers would be a good idea for this function anyway).

Reply:
looks like it will not, you need to iterate InitContainers also: 
https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/sti.go#L136

Reply:
Thanks Ben...  Updated and pushed.